### PR TITLE
Commvault Integration with Microsoft Azure Sentinel - V.1.0

### PR DESCRIPTION
**Pull request by Cv-securityIQ**
Change(s):
 - Created the Commvault Integration Solution that features the following:
    - Runbooks: Three of which are use cases for making API calls to Commvault/Metallic, and one for cycling access tokens
    - Analytic Rules: These query for manually created Sentinel Incidents with specific names and descriptions
    - A detailed Readme file that walks the user through creating the remainder of the integration assets through the GUI

   Reason for Change(s):
   - Initial integration PR.

   Version Updated:
   - Yes
   - Initial pull request.
   
   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/Commvault Security IQ/SolutionMetadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "publisherId": "azuresentinel",
```
Comment (v-rbajaj) [inline Solutions/Commvault Security IQ/SolutionMetadata.json:2]: Please change the publisherID to "CommvaultSecurityIQ"

File: Solutions/Commvault Security IQ/Package/mainTemplate.json [hunk 2]
```diff
@@ -0,0 +1,874 @@
...
+            "Automation-Account": {
+              "defaultValue": "Commvault-Automation-Account",
+              "type": "string"
+            },
+            "keyvaultName": {
+              "type": "string"
+            }
+          },
+          "variables": {
+            "location": "[[resourceGroup().location]",
```
Comment (v-rbajaj) [inline Solutions/Commvault Security IQ/Package/mainTemplate.json:357]: Please change the "location" to this value `"[parameters('workspace-location')]"`

Comment (Cv-securityIQ) [reply Solutions/Commvault Security IQ/Package/mainTemplate.json]: I've made the required changes and pushed them to the PR.

Comment (Cv-securityIQ) [reply Solutions/Commvault Security IQ/Package/mainTemplate.json]: The SolutionValidations is failing, can you please take a look?

File: Solutions/Commvault Security IQ/SolutionMetadata.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "publisherId": "CommvaultSecurityIQ",
```
Comment (v-rbajaj) [inline Solutions/Commvault Security IQ/SolutionMetadata.json:2]: This needs to be in lowercase.

Comment (Cv-securityIQ) [reply Solutions/Commvault Security IQ/SolutionMetadata.json]: Made the changes and pushed it to the PR, all the validations are passing now, Thanks!

**Review by v-rbajaj — APPROVED** [review]
looks good.

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**